src/components/ReviewConfirm/IncorporationDateTime.vue

```diff
@@ -0,0 +1,368 @@
+    this.selectPeriod = 'AM'
+
+    // Clear Future Effective from store / fee summary
+    this.setIsFutureEffective(false)
+    this.emitDateTime('')
+  }
+
+  /** Construct the Date Object for storage */
+  private constructDate (): void {
+    if (this.isFutureEffective) {
+      // Format the selected date string and create Date
+      const dateString = this.dateText.replace(/-/g, ', ')
+      const dateToValidate = new Date(dateString)
+
+      // Create references & Apply time period
+      let hours = +this.selectHour
+      const minutes = +this.selectMinute
+
+      if (this.selectPeriod === 'AM' && +this.selectHour === 12) {
+        dateToValidate.setDate(dateToValidate.getDate() - 1)
+        hours = +this.selectHour + 12
+      } else if (this.selectPeriod === 'PM' && +this.selectHour !== 12) {
+        hours = +this.selectHour + 12
+      }
+
+      // Apply selected hours and minutes
+      dateToValidate.setHours(hours, minutes)
+
+      // Set DateTime
+      this.emitDateTime(dateToValidate)
+    }
+    this.emitIsValidDateTime()
+  }
+
+  /** Parse the DateTime from store */
+  private deconstructDateTime (): void {
+    if (this.incorporationDateTime) {
+      // Set the chosen filing time option
+      for (let [key, val] of Object.entries(this.incorporationDateTime)) {
+        if (val === true) this.selectDate = key
+      }
+      // Reference Store and create date object
+      const effectiveDate = this.incorporationDateTime.futureEffectiveDate
+      const dateToParse = effectiveDate && new Date(effectiveDate)
+
+      if (dateToParse) {
+        // Parse the Date
+        const year = dateToParse.getFullYear()
+        const month = dateToParse.getMonth()
+        const day = dateToParse.getDate()
+
+        // Parse the Time and display DateTime
+        if (year && month && day) {
+          let hour = dateToParse.getHours()
+          const minute = dateToParse.getMinutes()
+          const amPm = hour >= 12 ? 'PM' : 'AM'
+
+          if (hour > 12) {
+            hour -= 12
+          } else if (hour === 0) {
+            hour = 12
+          }
+
+          this.dateText = `${year}-${month + 1}-${day}`
+          this.selectHour = hour.toLocaleString()
+          this.selectMinute = minute.toLocaleString('en-US', { minimumIntegerDigits: 2 })
+          this.selectPeriod = amPm
+        }
+      }
+    }
+  }
+
+  /** Validate the DateTime is within the allowed range */
+  private isValidDateTime (): boolean {
+    if (this.incorporationDateTime.futureEffectiveDate) {
+      const effectiveDate = this.incorporationDateTime.futureEffectiveDate
+      const dateToParse = new Date(effectiveDate)
+      const startDate = new Date()
+
+      const timeDiff = dateToParse.getTime() - startDate.getTime()
+      const timeDiffInMinutes = Math.floor(timeDiff / 1000 / 60)
+
+      // Time set must be more than 2 minutes and less than 10 days
+      return timeDiffInMinutes > 2 && timeDiffInMinutes < 14400
```

Comment:
Spec says "at least 2 min" and "at most 10 days", so these should be >= and <=.

https://preview.uxpin.com/6bd383a032304acc6ef186307cf186993fb02a39#/pages/127750603/simulate/no-panels

Reply:
ahh, good catch. Will update, thanks!